webcompat/issues.py

```diff
@@ -57,7 +57,10 @@ def report_public_issue(form):
     Returns a requests.Response object.
     """
     path = 'repos/{0}'.format(REPO_URI)
-    return proxy_request('post', path, data=json.dumps(unmoderated_issue()))
+    public_data = unmoderated_issue()
+    # We add action-needsmoderation label, so reviewers can filter out
+    public_data['labels'] = ['action-needsmoderation']
```

Comment:
I think eventually we will want to update https://github.com/webcompat/webcompat.com/blob/d56cd783366d725bcf2cf560723a590faf41e08f/webcompat/static/js/lib/models/label-list.js#L27, but this is fine for now -- the labels will appear on GitHub.

Reply:
Filed https://github.com/webcompat/webcompat.com/issues/3161 as a follow up for someone. Not urgent.